src/Microsoft.TemplateEngine.Cli/Alias/AliasManipulationResult.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Collections.Generic;
+
+namespace Microsoft.TemplateEngine.Cli.Alias
+{
+    internal class AliasManipulationResult
+    {
+        internal AliasManipulationResult(AliasManipulationStatus status)
+            : this(status, null, null)
+        {
+        }
+
+        internal AliasManipulationResult(AliasManipulationStatus status, string? aliasName, IReadOnlyList<string>? aliasTokens)
+        {
+            Status = status;
+            AliasName = aliasName;
+            AliasTokens = aliasTokens;
+        }
+
+        internal AliasManipulationStatus Status { get; }
```

Comment:
nit: does coding standard you are using dictate that internal classes has to have all members with internal modified. IMHO, having class internal and members public is slightly better approach.